0000-getting-started-cmake.md

```diff
@@ -5,7 +5,7 @@ aliases: [ "00-getting-started" ]
 
 In this getting started guide we will setup your machine to perform local CMake containerized hermetic and cached builds.
 
-1. Install `cmake-re` a drop in replacement for CMake :
+1. Install `cmake-re` as drop in replacement for CMake:
```

Comment:
I think this now reads that people can't use cmake directly afterwards, because it would be shadowed by cmake-re. I would change it back to "a drop in replacement for CMake" or remove it then, but I felt it was important that people had a short info to read what they would install.